lib/ansible/modules/source_control/gitlab_service.py

```diff
@@ -0,0 +1,377 @@
+        base_specs['argument_spec']['params'] = dict(required=False, type='dict', options=definitions[service])
+        module = AnsibleModule(**base_specs)
+    else:
+        module = stub_init
+
+    if not HAS_GITLAB_PACKAGE:
+        module.fail_json(msg=missing_required_lib("python-gitlab"), exception=GITLAB_IMP_ERR)
+
+    api_url = module.params['api_url']
+    validate_certs = module.params['validate_certs']
+    api_username = module.params['api_username']
+    api_password = module.params['api_password']
+    api_token = module.params['api_token']
+    project = module.params['project']
+    active = True
+    # active = module.params['active']
+    params = module.params['params']
+    events = module.params['events']
+
+    try:
+        git = Gitlab(url=api_url, ssl_verify=validate_certs, email=api_username, password=api_password, private_token=api_token, api_version=4)
+        # git.enable_debug() ?
+        git.auth()
+        project = git.projects.get(project)
+    except (gitlab.GitlabHttpError, gitlab.GitlabAuthenticationError, gitlab.GitlabGetError) as e:
+        module.fail_json(msg='Failed to connect to GitLab server: %s' % to_native(e))
+
+    try:
+        remote_service = project.services.get(service)
+    except gitlab.GitlabGetError as e:
+        module.fail_json(msg='No such a service %s' % service, exception=to_native(e))
+
+    ServicesHelper = GitLabServices(module, git)
+    if state == 'absent':
+        if not remote_service or not remote_service.created_at:
+            module.exit_json(changed=False, service={}, msg='Service not found', details='Service %s not found' % service)
+        else:
+            if module.check_mode:
+                module.exit_json(changed=True, service=remote_service.attributes)
+            try:
+                remote_service.delete()
+            except (gitlab.GitlabHttpError, gitlab.GitlabDeleteError) as e:
+                module.fail_json(msg='Failed to remove service %s' % service, exception=to_native(e))
+            else:
+                module.exit_json(changed=True, service=remote_service.attributes, result='Successfully deleted service %s' % service)
+
+    else:
+        if remote_service.created_at:
+            # update
+            try:
+                diff = {'before': str(remote_service.attributes)}
+                h = ServicesHelper.update(project, remote_service, active, params, events)
+                # if diff: fetch again
+                # diff['after'] = str(remote_service.attributes)
+            except gitlab.GitlabUpdateError as e:
+                module.fail_json(changed=False, msg='Could not update service %s' % service, exception=to_native(e))
+            else:
+                if h:
+                    module.exit_json(changed=True, service=remote_service.attributes, diff=diff, state='changed',
+                                     result='Successfully updated service %s' % service)
```

Comment:
Use `msg` instead of `result` attribute
```suggestion
                                     msg='Successfully updated service %s' % service)
```